topics/wordplate/index.md

```diff
@@ -0,0 +1,11 @@
+---
+topic: wordplate
+created_by: Vincent Klaiber
+display_name: WordPlate
+github_url: https://github.com/wordplate
+logo: wordplate.png
+released: March 18, 2015
+short_description: WordPlate is a modern WordPress stack built with Composer.
+url: https://wordplate.github.io/
+---
+WordPlate is a modern WordPress stack built with Composer. It is built on top of Laravel and Symfony components.
```

Comment:
Can this be expanded a bit to explain why I would use WordPlate? I've used WordPress before, but I'm wondering what "WordPress stack" means. Reading a bit on https://www.sitepoint.com/wordplate/ and https://managewp.org/articles/11109/wordplate-a-wordpress-boilerplate, there are a few points that I think could be rephrased and expanded here:

- WordPlate is a different way of setting up WordPress
- Lets you manage your WordPress site with Composer
- Helps with WordPress development with packages from Packagist

Reply:
Okay, what about this?

_WordPlate is a modern WordPress stack. WordPlate uses WordPress as a framework and it lets you install plugins and themes with Composer via WPackagist._

If you see any grammar issues, please let me know.

Reply:
@cheshire137 okay, what about this?

_WordPlate is a modern WordPress stack. WordPlate uses WordPress as a framework and it lets you install plugins and themes with Composer via WPackagist. WordPlate is built on top of Laravel and Symfony components._

If you see any grammar issues, please let me know.

Reply:
For the second sentence, maybe:

> WordPlate uses WordPress as a framework that lets you install plugins and themes with Composer via Packagist.

Otherwise, sounds good!

Reply:
Cool! Maybe we should mention WPackagist thought? https://wpackagist.org/

Reply:
I've pushed an update.

Reply:
> Maybe we should mention WPackagist

Yep, I agree!